internal/api/handlers/schedulers_handler.go

```diff
@@ -53,7 +54,12 @@ func ProvideSchedulersHandler(schedulerManager *scheduler_manager.SchedulerManag
 }
 
 func (h *SchedulersHandler) ListSchedulers(ctx context.Context, message *api.ListSchedulersRequest) (*api.ListSchedulersResponse, error) {
-	schedulerEntities, err := h.schedulerManager.GetSchedulersWithFilter(ctx, nil)
+	schedulerFilter := &filters.SchedulerFilter{
+		Name:    message.Name,
```

Comment:
WDYT about using "message.GetName()" instead of directly accessing the property? It's a good practice since these "GetX()" methods are auto-generated and take care of nil values and stuff. If you look at the other handlers, they're using this pattern too 🤟

Reply:
Good catch, thank you!